auth/oidc.go

```diff
@@ -103,6 +103,19 @@ type OIDCProvider struct {
 	DiscoveryURI            string   `json:"discovery_url,omitempty"`          // Non-standard discovery endpoints
 	DisableConfigValidation bool     `json:"disable_cfg_validation,omitempty"` // Bypasses config validation based on the OIDC spec.  Required for some OPs that don't strictly adhere to spec (eg. Yahoo)
 
+	// DisableCallbackState determines whether or not to maintain state between the
+	// Authentication (OAuth 2.0 Authorization) request and the callback. The default
```

Comment:
It isn't clear from this description that this applies to the auth code flow.  I think it would be preferable to describe which SG API endpoints this applies to, as opposed to the generic "Authentication (OAuth 2.0 Authorization) request"